app/livechat/server/lib/Livechat.js

```diff
@@ -49,6 +51,21 @@ export const Livechat = {
 		},
 	}),
 
+	addTyingListener(rid, callback) {
```

Comment:
You should do a search for 'Tying' --> Typing, looks like there are a couple spots where that shows up

Reply:
Thanks @dlassalle Fixed it ✅